src/components/Icons/LoadIndicator/LoadIndicatorIdle.jsx

```
@@ -0,0 +1,79 @@
+
+const LoadIndicatorIdle = (props) => {
+  const { color = "#175CD3", disabled, ...restProps } = props;
```

ESLint

[error] 'color' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)